examples/jsm/webxr/XRHandOculusMeshModel.js

```diff
@@ -11,7 +11,8 @@ class XRHandOculusMeshModel {
 		var loader = new FBXLoader();
 		const low = options && options.model === "lowpoly" ? "_low" : "";
 
-		loader.load( `../../models/fbx/OculusHand_${handedness === "right" ? "R" : "L"}${low}.fbx`, object => {
+		loader.setPath( path );
+		loader.load( `fbx/OculusHand_${handedness === "right" ? "R" : "L"}${low}.fbx`, object => {
```

Comment:
Why `fbx/`?

Reply:
I believe it's because they're currently stored under examples/models/fbx

Reply:
https://github.com/mrdoob/three.js/commit/7cb0a77f29fc4fd9e11485a235b82b27d7d8a31e